kaggle_environments/main.py

```diff
@@ -230,21 +303,26 @@ def http_request(request):
             del params[key]
 
     body = request.get_json(silent=True, force=True) or {}
-    req = parse_args({**params, **body})
-    resp = action_handler(req)
+    args = {**params, **body}
+    if "render" in args:
+        args["render"] = json.loads(args["render"])
+    args = parse_args(args)
+    if args.log_path is None:
+        args.log_path = log_path
+    resp = action_handler(args)
     return resp, 200, headers
 
 
 def main():
-    args = parser.parse_args()
+    args = parse_args(vars(parser.parse_args()))
```

Comment:
This ensures http-server mode receives all of the same args that cmd-line mode receives